app/settings.py

```diff
@@ -173,6 +174,12 @@
 MEDIA_ROOT = 'files'
 MEDIA_URL = '/files/'
 
+if os.environ.get('DROPBOX_OAUTH2_TOKEN', False):
+    DEFAULT_FILE_STORAGE = 'storages.backends.dropbox.DropBoxStorage'
+    DROPBOX_OAUTH2_TOKEN = os.environ.get('DROPBOX_OAUTH2_TOKEN', False)
+    assert DROPBOX_OAUTH2_TOKEN, 'Set DropBox oauth2 token'
```

Comment:
This entire block doesn't run when this assertion can be triggered.

Reply:
True, left the assertion from a previous version. Will remove it